lib/common/environment.dart

```diff
@@ -139,6 +139,16 @@ class Environment {
   static String get branchKey => dotenv.env['BRANCH_KEY'] ?? '';
 
   static String get mixpanelKey => dotenv.env['MIXPANEL_KEY'] ?? '';
+
+  static String get cloudFlareImageUrlPrefixMainnet =>
```

Comment:
It's not configurated by mainnet/testnet. We should only use one env: `CLOUD_FLARE_IMAGE_URL_PREFIX`.